content/blog/2022-01-15-moving-local-experiments-to-the-cloud-with-iterative-terraform-provider.md

```diff
@@ -196,47 +197,57 @@ Take a look at the script. In the simplest scenario, all we need to do on a new
 machine to run the training script is to set up the Python environment with
 required libraries. If you simply want to train
 your model on a machine with more memory, this may be enough. However, if you
-want your training code to leverage GPUs, the script will look a bit more
-complex.
+want your training code to leverage GPUs, the script will look a bit different.
 
 ## Training with GPU
 
-Setting up the environment to leverage GPU devices is a bit involved as you need
-to make sure you have the required drivers, NVIDIA-docker, and CUDA.
+There are several ways you can leverage GPU devices on a remote machine. You can install all 
+the required drivers and dependencies "manually" via a script, you can use an existing Docker image 
+or build your own, or, in the case of AWS, you can take advantage of an existing pre-configured Deep Learning AMI. 
+In this tutorial, we'll use an AMI, as it is the quickest way to start using GPUs on an AWS EC2 machine.
 
 ```hcl
 resource "iterative_task" "tpi-examples-gpu" {
-    name      = "tpi-examples-gpu"
+    name      = "ami-gpu-example"
     cloud     = "aws"
     region    = "us-east-2"
     machine   = "l+k80"
+    disk_size = "130"
+    image     = "ubuntu@898082745236:x86_64:Deep Learning AMI (Ubuntu 18.04) Version 54.0"
```

Comment:
could we use latest/20.04? And one of the base images (without conda)? Something with a smaller disk size requirement?

Reply:
Why? That doesn't really change anything in regards to the example on how to use TPI.

Reply:
may potentially reduce spin-up time?

Reply:
purged, using `image = "nvidia"` now